### ISSUE TITLE
Add Dapps on IPFS guide

### ISSUE THREAD

**Issue by 2color** [ipfs/ipfs-docs#1791]
This is a placeholder issue to track the work on a Dapps on IPFS guide

From a high level it should cover:
- Reasons to use IPFS for Dapps? (Verifiability/Trustlessness, resilience, regulatory compliance)
- Dapp architectures, how they use IPFS
- Current landscape for publishing and loading of CIDs
- Different ways to retrieve
- Naming systems and how they apply to Dapps on IPFS (ENS, DNSLink, IPNS)

**Comment by SgtPooki** [comment]
@2color Is this ultimately a tutorialized version of the state of dApps on IPFS blog post?

**Comment by 2color** [comment]
Yeah a broken down copied version. But not a high priority at all
